bitnami/elasticsearch/templates/coordinating-statefulset.yaml

```diff
@@ -161,7 +161,7 @@ spec:
             {{- end }}
           {{- end }}
           ports:
-            - name: http
+            - name: {{ if .Values.security.enabled }}https{{ else }}http{{ end }}
```

Comment:
It seems the same change is needed in several places, Could you move this logic to the `_helpers.tpl` file?